### ISSUE TITLE
Humble Bundle: "Failed read Humble App config" during library update

### ISSUE THREAD

**Issue by darklinkpower** [JosefNemec/PlayniteExtensions#151]
**Check for existing issue**
None found

**Describe the bug**
Error "Failed to import games from Humble. Failed read Humble App config" happens during library update

**To Reproduce**
Try to update library of Humble App

**Screenshots**
![image](https://user-images.githubusercontent.com/1389286/156268255-3c902f05-4876-46a5-b33f-110c06851a4c.png)

**Diagnostics ID**
baaeda12-276e-4664-aeb1-a428318af34f

**Comment by darklinkpower** [comment]
Checking the logs, it might be related to the app currently downloading games during the attempted library update

![image](https://user-images.githubusercontent.com/1389286/156268477-6620f55b-4ba9-4005-86f9-18659bad28c7.png)

```
01-03 17:48:22.570|ERROR|HumbleLibrary#HumbleLibrary:Failed read Humble App config.
Newtonsoft.Json.JsonReaderException: Input string '0.011246426621526353' is not a valid integer. Path 'game-collection-4[70].downloadPercentage', line 3880, position 45.
   en Newtonsoft.Json.JsonTextReader.ParseReadNumber(ReadType readType, Char firstChar, Int32 initialPosition)
   en Newtonsoft.Json.JsonTextReader.ParseNumber(ReadType readType)
   en Newtonsoft.Json.JsonTextReader.ReadNumberValue(ReadType readType)
   en Newtonsoft.Json.JsonTextReader.ReadAsInt32()
   en Newtonsoft.Json.JsonReader.ReadForType(JsonContract contract, Boolean hasConverter)
   ```

**Comment by darklinkpower** [comment]
I can't reproduce it again but I think the error should be solved by changing the type from `int` to `double` or `long`

https://github.com/JosefNemec/PlayniteExtensions/blob/0de9478f2de3a8c7aa4764a5151737562cad2762/source/Libraries/HumbleLibrary/Models/HumbleApp.cs#L62

**Comment by JosefNemec** [comment]
Fixed in 3.3
